crates/core_types/src/network/query.rs

```diff
@@ -13,6 +13,14 @@ pub enum GetLinksNetworkQuery {
     Links(GetLinksQueryConfiguration),
 }
 
+// The data returned by a remote node on responding to a get_links query
+#[derive(Debug, Serialize, Deserialize, PartialEq, DefaultJson, Clone)]
+pub struct GetLinkFromRemoteData {
+    pub link_add_address: Address,
```

Comment:
From what I can tell this name is probably not correct, because this is either the address of the entry that added the link or the address of the entry that removed it in the case that the link was removed and the `LinksStatusRequestKind` was `All` or `Deleted`  Is that correct?

If so then we should rename this to `linking_action_address` or something else that makes it clear that this field can hold the address of either type of entry.